src/lib/components/modal/bottomModal/index.tsx

```diff
@@ -70,7 +70,7 @@ export default ({
             </button>
           )}
         </div>
-        <div className={styles.content}>{children}</div>
+        <div className={styles.content}><div className="pb48">{children}</div></div>
```

Comment:
This could be done with one div if I understand correctly.

Reply:
You're right 🤦  I thought the styles would be ovewritten but it's actually storybook doing that.